packages/ripple-binary-codec/src/utils.ts

```diff
@@ -0,0 +1,149 @@
+// Don't know why this comes from NodeJS as it is valid in the browser
+import TypedArray = NodeJS.TypedArray
+
+//
+/**
+ * Writes value to array at the specified offset. The value must be a valid unsigned 8-bit integer.
+ * @param array Uint8Array to be written to
+ * @param value Number to be written to array.
+ * @param offset plus the number of bytes written.
+ */
+export function writeUInt8(
+  array: Uint8Array,
+  value: number,
+  offset: number,
+): void {
+  value = Number(value)
+  array[offset] = value
+}
+
+/**
+ * Writes value to array at the specified offset as big-endian. The value must be a valid unsigned 16-bit integer.
+ * @param array Uint8Array to be written to
+ * @param value Number to be written to array.
+ * @param offset plus the number of bytes written.
+ */
+export function writeUInt16BE(
+  array: Uint8Array,
+  value: number,
+  offset: number,
+): void {
+  value = Number(value)
+
+  array[offset++] = value >>> 8
+  array[offset++] = value
+}
+
+/**
+ * Writes value to array at the specified offset as big-endian. The value must be a valid unsigned 32-bit integer.
+ * @param array Uint8Array to be written to
+ * @param value Number to be written to array.
+ * @param offset plus the number of bytes written.
+ */
+export function writeUInt32BE(
+  array: Uint8Array,
+  value: number,
+  offset: number,
+): void {
+  array[offset] = (value >>> 24) & 0xff
+  array[offset + 1] = (value >>> 16) & 0xff
+  array[offset + 2] = (value >>> 8) & 0xff
+  array[offset + 3] = value & 0xff
+}
+
+/**
+ * Reads an unsigned, big-endian 16-bit integer from buf at the specified offset.
```

Comment:
`buf` was the old name for the parameter I think
```suggestion
 * Reads an unsigned, big-endian 16-bit integer from array at the specified offset.
```

Reply:
Please apply this to all the docstrings in this file

Reply:
(Maybe better to say `from the array` instead of `from array` since array can be confused for a word instead of the name of a parameter as it's being used in this case

Reply:
Fixed in aa343f183a934ea60f37fec6f00d16a7b20ce89b